src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -513,6 +523,19 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps, GraphSetting
             load relevant to the other edges. Animation can be CPU intensive.
           </div>
         )
+      },
+      {
+        id: 'rank',
+        labelText: 'Rank',
+        isChecked: rank,
+        onChange: toggleRank,
+        tooltip: (
+          <div style={{ textAlign: 'left' }}>
+            Rank graph nodes based on configurable criteria such as 'number of inbound edges'. These rankings can be
+            used in the graph find/hide feature to help highlight the most important workloads, services, and
+            applications. Rankings are normalized to fit between 1..100 and nodes may tie with each other in rank.
+          </div>
+        )
       }
```

Comment:
This looks good using the new approach.  But this option should be placed alphabetically in the `Show` section.

I needed more from the tooltip text.  I see the ranks are 1..100 but I don't know if 1 is highest or lowest.  Maybe continue the example, "for example, the node with the highest number of inbound edges would be ranked 1". (or is it 100 :-)).

Reply:
Let me know if it's more clear now